components/Footer/Footer.tsx

```diff
@@ -102,6 +104,12 @@ const Appendix = () => (
             >
                 Anti-Slavery Policy
             </a>
+            <a
+                className={styles.appendixLink}
+                href="https://torchbox.com/environment-policy/"
+            >
+                Sustainability Policy
+            </a>
```

Comment:
It looks like we'll need to choose a different styling approach for the appendix links, as they are now wrapping in an odd way on mobile devices: 

<img width="551" alt="Screenshot 2023-06-28 at 10 08 12" src="https://github.com/torchbox/careers/assets/18164832/0df860c9-c82f-474a-b8e1-5b964bcc8b5f">

I think a flex container with flex-wrap & gap would be the way to go here: https://codepen.io/jhancock532/pen/zYMZZNv

Reply:
Yep - good spot. I will update this on torchbox.com too.